### PR TITLE
Add more users agents of bots found in the wild

### PR DESCRIPTION
**Pull request by komapa**
These are real-world bots that we see daily and we have added manually to account for the fact that they are not present in this repo. I hope this is helpful for others as well.

### CODE REVIEW

**Comment by monperrus** [comment]
Thanks for your contribution. There are problems, see CI logs.

**Comment by komapa** [comment]
Should be ready for review again. Thanks!

File: crawler-user-agents.json [hunk 1]
```diff
@@ -4161,13 +4161,6 @@
     ],
     "url": "https://developer.amazon.com/support/amazonbot"
   },
-  {
-    "pattern": "Googlebot-Compatible",
-    "addition_date": "2020/03/02",
-    "instances": [
-      "Mozilla/5.0 (X11; U; Linux x86_64; de; rv:1.9.2.8) Googlebot-Compatible Gecko/20100723 Ubuntu/10.04 (lucid) Firefox/3.6.8"
-    ]
-  },
```
Comment (komapa) [inline crawler-user-agents.json]: 👍

**Comment by komapa** [comment]
Bump!

**Comment by monperrus** [comment]
Thanks a lot!

**Pull request merged**